mmrazor/structures/estimator/op_spec_counters/base_counter.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from abc import ABCMeta, abstractclassmethod
+
+
+class BaseCounter(object, metaclass=ABCMeta):
```

Comment:
Point that `XXModuleCounter` is responsible for `XXModule`, which could refers to flops_params_counter::`get_counter_type()`.

Reply:
Done.